configure.ac

```diff
@@ -1,7 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([libdfp],[1.0.14],[https://github.com/libdfp/libdfp/issues])
+AC_INIT([libdfp],[1.0.15],[https://github.com/libdfp/libdfp/issues])
```

Comment:
Let's save this change for the formal commit which introduces version 1.0.15. Likewise, ./Versions.def should include this, I don't think I see it here.

Aside from some rebase conflicts, this looks OK.